web/components/config/VideoVariantForm.tsx

```diff
@@ -108,17 +108,23 @@ export const VideoVariantForm: FC<VideoVariantFormProps> = ({
   });
   return (
     <div className={classes}>
-      <p className="description">
-        <a
-          href="https://owncast.online/docs/video?source=admin"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn more
-        </a>{' '}
-        about how each of these settings can impact the performance of your server.
-      </p>
-
+      <div className="video-varient-alert">
+        <Alert
+          description="Learn more about how each of these settings can impact the performance of your server."
+          type="info"
+          action={
+            <a
+              href="https://owncast.online/docs/video?source=admin"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <Button size="small" type="text">
+                Visit the link
```

Comment:
Just a small nit. Instead of "Visit the link" maybe "Learn more" and you could change the description to something like "See how each of these settings"... Talking about links just seemed weird :)

Reply:
Hi there! Thanks a lot for your feedback! I will change it :)